python/ray/data/read_api.py

```diff
@@ -559,6 +563,7 @@ def read_parquet(
     shuffle: Union[Literal["files"], None] = None,
     include_paths: bool = False,
     file_extensions: Optional[List[str]] = None,
+    concurrency: Optional[int] = None,
```

Comment:
Are we planning on adding the `concurrency` parameter to all of the other read APIs? If not, why are we special casing `read_parquet`?

Reply:
yes, it's on PR description.

Reply:
Ah, my bad. Totally missed that

Reply:
No worries.